build.gradle

```diff
@@ -398,7 +398,6 @@ task createApiDocsRedirect {
 }
 
 task publishDocs {
-  dependsOn 'closeAndReleaseRepository'
```

Comment:
Looks like this was introduced in https://github.com/googleapis/gax-java/pull/302